search/views/bizdev_search_core_users.view.lkml

```diff
@@ -1,46 +1,13 @@
 view: bizdev_search_core_users {
   derived_table: {
```

Comment:
Does this still need to be a derived table? You could add `mozdata.search.desktop_mobile_search_clients_monthly` to the lookml-generator and use the usual flow to have this file be auto-generated

Reply:
That sounds like a better thing to do. I will create a PR and send it your way